File: engine/engine-model/src/main/scala/tools/packet/event/CoconutScore.scala
package tools.packet.event

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class CoconutScore(private var _firstTeam: Int, private var _secondTeam: Int) extends PacketInput {
  def firstTeam: Int = _firstTeam

  def secondTeam: Int = _secondTeam

  override def opcode(): SendOpcode = SendOpcode.COCONUT_SCORE
}

File: engine/engine-model/src/main/java/tools/packet/event/CoconutScore.java
package tools.packet.event;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record CoconutScore(Integer firstTeam, Integer secondTeam) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.COCONUT_SCORE;
   }
}